src/lime/extension/Main.hx

```diff
@@ -554,38 +567,33 @@ class Main {
 		
 		if (!hasProjectFile || !isProviderActive) return;
 		
-		var projectFile = getProjectFile ();
-		var buildConfigFlags = getBuildConfigFlags ();
-		var targetFlags = getTargetFlags ();
-		
-		var commandLine = "lime " + getCommandArguments ("display").join (" ");
+		var commandLine = limeCommand + " " + getCommandArguments ("display").join (" ");
 		commandLine = StringTools.replace (commandLine, "-verbose", "");
-		
-		//trace ("Running display command: " + commandLine);
-		
-		try {
-			
-			ChildProcess.exec (commandLine, { cwd: workspace.rootPath }, function (err, stdout:Buffer, stderror) {
-				
-				try {
-					
-					displayArgumentsProvider.update (stdout.toString ());
-					
-				} catch (e:Dynamic) {
-					
-					trace ("Error running display command: " + commandLine);
-					trace (e);
-					
-				}
-				
-			});
-			
-		} catch (e:Dynamic) {
```

Comment:
I don't think `ChildProcess.exec()` can actually throw an exception (as opposed to `execSync`), so previously, failure was really completely silent, not even any output in the dev console. Now we check the exit code in the callback instead.